docs/exposing-eventlisteners.md

```diff
@@ -0,0 +1,58 @@
+# Exposing EventListeners Externally
+
+By default, `ClusterIP` services such as the EventListener sink are accessible 
+within the cluster. There are a few ways of exposing it so that external services
+can talk to it:
+
+
+## Using an Ingress
+
+You can use an Ingress resource to expose the EventListener. The [`create-ingress`](./create-ingress.yaml)
+Tekton task can help setup an ingress resource using self-signed certs.
+
+**Note**: If you are using a cloud hosted Kubernetes solution such as GKE, the built-in ingress will not work
+with `ClusterIP` services. Instead, you can use the Nginx Ingress based approach below.
+
+
+## Using Nginx Ingress
+
+The following instructions have been tested on GKE cluster running version `1.13.7-gke.24`.
+Instructions for installing nginx Ingress on other Kubernetes services can be found [here](https://kubernetes.github.io/ingress-nginx/deploy/).
+
+1. First, install Nginx ingress controller:
```

Comment:
Nit: Indentation is off? Everything is step 1.

Reply:
done!